bot/utils/services.py

```diff
@@ -47,7 +47,14 @@ async def send_to_paste_service(contents: str, *, extension: str = "") -> Option
             continue
         elif "key" in response_json:
             log.info(f"Successfully uploaded contents to paste service behind key {response_json['key']}.")
-            return URLs.paste_service.format(key=response_json['key']) + extension
+
+            paste_link = URLs.paste_service.format(key=response_json['key']) + extension
+
+            if extension == '.py':
+                return paste_link
```

Comment:
Does py need to be special cased here? the noredirect shouldn't change anything for it and the util isn't used anywhere where the compactness would matter

Reply:
I guess not, but I still think it's nice to account for it.